pkg/google/gke/pricing_map.go

```diff
@@ -143,14 +135,20 @@ func NewMachineTypePricing() *FamilyPricing {
 	}
 }
 
+type StoragePrices struct {
+	ProvisionedSpaceGiB float64
+	Throughput          float64
+	IOops               float64
```

Comment:
The names are subject to change once I investigate what the returned metadata is for disks. I'm also partial to `IOops` as it leans in on the "oops" bit

Reply:
+1 I'm used to `IOps` in general, not fused on any of them fwiw.